clients/bigquery/bigquery_test.go

```diff
@@ -34,53 +34,6 @@ func (b *BigQueryTestSuite) TestTableRelName() {
 	}
 }
 
-func TestFullyQualifiedName(t *testing.T) {
```

Comment:
I added tests for `FullyQualifiedName` in https://github.com/artie-labs/transfer/pull/466 so these tests are redundant.